src/pages/carriere/partners.jsx

```diff
@@ -13,25 +14,19 @@ const PartnerIndexPage = ({ data }) => {
     <ContentfulPage page={page}>
       <Markdown>{page.content.content}</Markdown>
       <PartnerList>
-        <div className="list">
+        <Grid stretched doubling stackable columns={4}>
           {partners.map(partner => (
-            <Partner key={partner.node.id} partner={partner.node} />
+            <Grid.Column>
+              <Partner key={partner.node.id} partner={partner.node}/>
+            </Grid.Column>
           ))}
-        </div>
+        </Grid>
       </PartnerList>
     </ContentfulPage>
   );
 };
 
 const PartnerList = styled.div`
```

Comment:
You forgot to remove the constant. Now it's just an empty styled component. Also remove it from in the JSX.